lib/extractor/layer.ts

```diff
@@ -148,6 +263,10 @@ function isManifestFile(name: string): boolean {
   return name === "manifest.json";
 }
 
+function isImageIndexFile(name: string): boolean {
```

Comment:
what's the ```Image``` in the function name here?

Reply:
This is referring to the `image index` file (documentation reference), which is a root manifest: https://github.com/opencontainers/image-spec/blob/master/image-index.md